### ISSUE TITLE
Skipping deserializing 

### ISSUE THREAD

**Issue by anfractuosity** [serde-rs/serde#294]
With the following I get an unknown field error:

```
#[derive(Serialize, Deserialize, Debug)] 
struct Point { 
    x: i32, 
    y: i32, 
      
    #[serde(skip_deserializing)] 
    z: Option<Arc<Mutex<i32>>>, 
} 
```

```
src/main.rs:17:13: 17:31 error: unknown serde field attribute `skip_deserializing`
src/main.rs:17     #[serde(skip_deserializing)]
```

Just wondering why that is?

I'm using the latest Rust nightly and the following in Cargo.toml
```
[dependencies]
serde = "*"
serde_json = "*"
serde_macros = "*"
```

cheers
Chris

**Comment by dtolnay** [comment]
This was merged in 7 days ago (#265) and has not been in a release yet, so `serde_macros = "*"` will not pick it up. You can try having Cargo pull in serde_macros from git if you are feeling brave.

**Comment by anfractuosity** [comment]
Awesome! I've give that a shot cheers
